cranelift/codegen/src/isa/x64/inst/args.rs

```diff
@@ -291,13 +301,23 @@ impl SseOpcode {
             SseOpcode::Insertps | SseOpcode::Roundss | SseOpcode::Roundsd => SSE41,
         }
     }
+
+    /// Which `InstructionSet` is the first supporting this opcode?
+    pub(crate) fn src_size(&self) -> u8 {
```

Comment:
This function seems to be unused, can we remove it?

Reply:
This is used in mod.rs and the impact comes during testing. We have no way to specify a 32-bit gpr register so for example instead of RegMem::reg(eax) what is being done in other gpr instruction tests is to use RegMem::reg(rax) but then create a parameter that sets the is_64 bool to false. So to print out ether movl vs movq the reg used is first passed in rbx but the bool determines whether rax or eax is printed. 

>     // Mov_R_R
>     insns.push((
>         Inst::mov_r_r(false, rbx, w_rsi),
>         "89DE",
>         "movl    %ebx, %esi",
>     ));
>     ...
>     insns.push((
>         Inst::mov_r_r(true, rbx, w_rsi),
>         "4889DE",
>         "movq    %rbx, %rsi",
>     ));

I too started with this parameter for xmm_mov_rm_r when I realized I needed to have the distinction for proper testing, but it was somewhat redundant and heavyweight to add this parameter everywhere as the opcode itself already specifies the operand size. That's the point of this function but right now it is only called in one place in mod.rs:

![image](https://user-images.githubusercontent.com/45402135/85072498-12e8ff80-b16e-11ea-842a-4b5fc129b7b5.png)

Reply:
Ok, thanks for the explanation! Didn't see this use there.